### PR TITLE
Finesse proof posting buttons.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

It makes the first step (posting to whatever service) the primary button instead of a link, and then it shows the “OK posted! Check for it!” after you've pressed the button for the first action.

### CODE REVIEW

File: shared/profile/post-proof/container.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   lifecycle({
     componentDidMount() {
-      // Facebook proof checking gets enabled after they click continue.
-      if (this.props.platform === 'facebook') {
+      // Active the proof check after they've completed the first step for these services.
+      if (['facebook', 'twitter', 'reddit', 'github', 'hackernews'].includes(this.props.platform)) {
```
Comment (adamjspooner) [inline shared/profile/post-proof/container.js]: This could get out of hand. I'm open to suggestions.

**Comment by adamjspooner** [comment]
@cecileboucheron, how does this look?
screen shot 2018-07-31 at 09 40 52

**Comment by adamjspooner** [comment]
@keybase/react-hackers and @cecileboucheron, bump.

**Comment by cecileboucheron** [comment]
I'd just kill this verbose and do:
`Post your proof to Facebook, and make sure the post is _public_.`
WDYT?

**Comment by adamjspooner** [comment]
I think a brief explanation on why might alleviate any concerns. What about:

```
Post your proof to Facebook. Make sure it's _public_, or we won't be able to read it.
```

**Comment by adamjspooner** [comment]
Are we dropping the GIF?

**Comment by cecileboucheron** [comment]
Sure. No I would keep the GIF but some time we should make a better one...

**Pull request merged**